<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabAppli">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="toolTipDuration" stdset="0">
       <number>-1</number>
      </property>
      <widget class="QWidget" name="marketPlace">
       <attribute name="title">
        <string>MarketPlace</string>
       </attribute>
       <widget class="QTabWidget" name="marketPlaceLesProduits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>760</width>
          <height>420</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <widget class="QWidget" name="alpha">
         <attribute name="title">
          <string>A-Z</string>
         </attribute>
         <widget class="QScrollArea" name="listLesProduitsAlpha">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>740</width>
            <height>360</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>738</width>
             <height>358</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="croissant">
         <attribute name="title">
          <string>Prix Croissant</string>
         </attribute>
         <widget class="QScrollArea" name="listLesProduitsCroiss">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>740</width>
            <height>360</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_4">
           <property name="geometry">
            <rect>
             <x>-640</x>
             <y>0</y>
             <width>738</width>
             <height>358</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="decroissant">
         <attribute name="title">
          <string>Prix Décroissant</string>
         </attribute>
         <widget class="QScrollArea" name="listLesProduitsDecrois">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>740</width>
            <height>360</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_5">
           <property name="geometry">
            <rect>
             <x>-640</x>
             <y>0</y>
             <width>738</width>
             <height>358</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QLineEdit" name="tagAChercher">
        <property name="geometry">
         <rect>
          <x>172</x>
          <y>20</y>
          <width>160</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="actionRechercher">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>20</y>
          <width>140</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLabel" name="rechercheProduit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>140</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher Produit : </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="monCompte">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Mon Compte</string>
       </attribute>
       <widget class="QTabWidget" name="tabMonCompte">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>760</width>
          <height>440</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="mesStats">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Mes Statistiques</string>
         </attribute>
         <widget class="QTabWidget" name="statLesCommentaires">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>9</y>
            <width>740</width>
            <height>390</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="eTQVendeur">
           <attribute name="title">
            <string>En tant que Vendeur</string>
           </attribute>
           <widget class="QScrollArea" name="listCommentairesV">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>720</width>
              <height>340</height>
             </rect>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_7">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>718</width>
               <height>338</height>
              </rect>
             </property>
             <widget class="QLabel" name="noteMoyDernierMois_ven">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>221</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Note moyenne du dernier mois : </string>
              </property>
             </widget>
             <widget class="QLabel" name="v_noteMoyDernierMois_ven">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>10</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>9.99</string>
              </property>
             </widget>
             <widget class="QLabel" name="noteMoyPeriode_ven">
              <property name="geometry">
               <rect>
                <x>420</x>
                <y>10</y>
                <width>251</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Note moyenne de la période active :</string>
              </property>
             </widget>
             <widget class="QLabel" name="v_noteMoyPeriode_ven">
              <property name="geometry">
               <rect>
                <x>680</x>
                <y>10</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>9.99</string>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="eTQAcheteur">
           <attribute name="title">
            <string>En tant qu'Acheteur</string>
           </attribute>
           <widget class="QScrollArea" name="listCommentairesA">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>720</width>
              <height>340</height>
             </rect>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_8">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>718</width>
               <height>338</height>
              </rect>
             </property>
             <widget class="QLabel" name="noteMoyDernierMois_ach">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>221</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Note moyenne du dernier mois : </string>
              </property>
             </widget>
             <widget class="QLabel" name="v_noteMoyDernierMois_ach">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>10</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>9.99</string>
              </property>
             </widget>
             <widget class="QLabel" name="noteMoyPeriode_ach">
              <property name="geometry">
               <rect>
                <x>420</x>
                <y>10</y>
                <width>251</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Note moyenne de la période active :</string>
              </property>
             </widget>
             <widget class="QLabel" name="v_noteMoyPeriode_ach">
              <property name="geometry">
               <rect>
                <x>680</x>
                <y>10</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>9.99</string>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="mesCoord">
         <attribute name="title">
          <string>Mes Coordonnées</string>
         </attribute>
         <widget class="QLabel" name="l_id">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>ID Utilisateur :</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_name">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>67</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_surname">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>67</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Prénom :</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_pseudo">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>67</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Pseudo :</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_adresse">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>67</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Adresse :</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_mdp">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>170</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Mot de passe :</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_tel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>200</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Téléphone :</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_ddn">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Date de naissance :</string>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>0</y>
            <width>750</width>
            <height>410</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="l_no_carte">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>20</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Numéro de Carte :</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_dexp">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>50</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Date d'expiration :</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_crypt">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>80</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Cryptogramme :</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_id">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>20</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_name">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>50</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_surname">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>80</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;surname&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_pseudo">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>110</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;pseudo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_adresse">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>140</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;adresse&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_mdp">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>170</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;mdp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_tel">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>200</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;tel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_ddn">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>230</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;ddn&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_no_carte">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>20</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;no_carte&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_dexp">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>50</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;dexp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="v_crypt">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>80</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;crypt&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="mesProduits">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Mes Produits</string>
       </attribute>
       <widget class="QScrollArea" name="listMesProduits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>760</width>
          <height>440</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>758</width>
           <height>438</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="lesLitiges">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Les Litiges</string>
       </attribute>
       <widget class="QScrollArea" name="listLesLitiges">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>760</width>
          <height>440</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>758</width>
           <height>438</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="mesCommandes">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Mes Commandes</string>
       </attribute>
       <widget class="QScrollArea" name="listMesCommandes">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>760</width>
          <height>440</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_9">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>758</width>
           <height>438</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnexion">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Authentification</string>
    </property>
    <addaction name="actionConnexion"/>
    <addaction name="actionInscription"/>
    <addaction name="actionDeconnexion"/>
   </widget>
   <widget class="QMenu" name="menuAjouter">
    <property name="title">
     <string>Ajouter</string>
    </property>
    <addaction name="actionAjouter_Produit"/>
   </widget>
   <addaction name="menuConnexion"/>
   <addaction name="menuAjouter"/>
  </widget>
  <action name="actionEn_tant_que_Vendeur">
   <property name="text">
    <string>En tant que Vendeur...</string>
   </property>
  </action>
  <action name="actionEn_tant_du_Acheteur">
   <property name="text">
    <string>En tant qu'Acheteur</string>
   </property>
  </action>
  <action name="actionConnexion">
   <property name="text">
    <string>Connexion</string>
   </property>
  </action>
  <action name="actionInscription">
   <property name="text">
    <string>Inscription</string>
   </property>
  </action>
  <action name="actionAjouter_Produit">
   <property name="text">
    <string>Ajouter Produit</string>
   </property>
  </action>
  <action name="actionDeconnexion">
   <property name="text">
    <string>Deconnexion</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tagAChercher</tabstop>
  <tabstop>marketPlaceLesProduits</tabstop>
  <tabstop>listLesProduitsCroiss</tabstop>
  <tabstop>listLesProduitsDecrois</tabstop>
  <tabstop>listLesProduitsAlpha</tabstop>
  <tabstop>tabMonCompte</tabstop>
  <tabstop>statLesCommentaires</tabstop>
  <tabstop>actionRechercher</tabstop>
  <tabstop>listLesLitiges</tabstop>
  <tabstop>listCommentairesV</tabstop>
  <tabstop>tabAppli</tabstop>
  <tabstop>listCommentairesA</tabstop>
  <tabstop>listMesCommandes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
